src/dir.rs

```diff
@@ -376,6 +416,11 @@ impl Dir {
     }
 
     /// Returns metadata of an entry in this directory
+    ///
+    /// Note that this method does not resolve symlinks by default, so you may have to call
```

Comment:
Returns metadata of the symlink itself rather than destination.

Reply:
Duly noted, thanks! I've clarified the docs to hopefully make it more clear to future readers.